vocabularies/survey-type.ttl

```diff
@@ -172,128 +83,23 @@ svytp:physical-geology a skos:Concept ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "em" ;
-    skos:prefLabel "Electromagnetic"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:geochemistry a skos:Concept ;
-    skos:definition "A survey where the chemical properties of rocks, soil, or other geological material is assessed to determine mineralogical and/or elemental composition."@en ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "gc" ;
-    skos:prefLabel "Geochemistry"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:photographic a skos:Concept ;
-    skos:definition "A survey using standard photographic equipment for image capture without intention for spectral analysis. Including land-based geological photography."@en ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "ph" ;
-    skos:prefLabel "Photographic"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:radiometric a skos:Concept ;
-    skos:altLabel "Gamma Logs"@en,
-        "Gamma-Ray Spectrography"@en ;
-    skos:definition "A survey where the emissions of natural gamma-rays are recorded to determine the quantity of radioelements (Potassium, Uranium, Thorium)."@en ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "rm" ;
-    skos:prefLabel "Radiometric"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:spectral a skos:Concept ;
-    skos:altLabel "hyperspectral"@en ;
-    skos:definition "A survey capturing image information and intensity-per-wavelength-band to be used for spectral analysis e.g. ASTER, MODIS."@en ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "sp" ;
-    skos:prefLabel "Spectral"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
 svytp:spatial a skos:Concept ;
+    skos:altLabel "Location Survey"@en ;
     skos:definition "A survey measuring the spatial characteristics of an object or region."@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
     skos:notation "st" ;
     skos:prefLabel "Spatial"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:gravity a skos:Concept ;
-    skos:definition "A survey where the variation of the Earth's gravity field in measured to determine the distribution of density in the region"@en ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "gv" ;
-    skos:prefLabel "Gravity"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:magnetic a skos:Concept ;
-    skos:altLabel "aeromagnetics"@en ;
-    skos:definition "A survey where the variations of the Earth's magnetic field is measured to determine the distribution of magnetic minerals (predominately magnetite)"@en ;
+    
+svytp:spectral a skos:Concept ;

```

Comment:
The remote sensing community would be puzzled by this categorisation. Also, in observation there are a bunch of things that don't fit this definition and, arguably, aren't "spectral' surveys. All are resolved if we simply call these remote sensing surveys?

Reply:
lidar and radar would be Location Survey i think in this classification.  Although i would be happy to resolve these both into Remote sensing if others think it makes more sense?

Reply:
As per above comment. some concepts need to be moved to a spatial (location survey) collection. This creates a logical link through to survey type in line with the other collections.

The inclusion of radar and lidar in that collection was a mistake that has lead people to a false idea that spectral was intended to be synonymous with remote sensing. Spectral and Spatial are the data types here, the survey method satellite, airborne etc would inform you on if that data was remotely sensed

Reply:
Thanks Vance, i think i am more comfortable with having the spectral / location survey as current.  Means we can separate geophysical activities and imagining (spectral) activities and spatial (lidar etc).  

Bu having spectral as a type means we can link between hylog and hymap surveys as discussed with Roger
Survey Type:                      Spectral                                             Spectral
Survey Method:                Airborne                                            Laboratory
Observation Type:           Spectral                                             Spectral                                             
Observation Method:     Multispectral                                    Hyperspectral
Instrument:                        ASTER                                                 Hylogger